client/Components/description.jsx

```diff
@@ -49,6 +50,10 @@ class Description extends React.Component {
       });
   };
 
+  openModal(e) {
+    e.preventDefault();
+    this.props.clickModal(e);
+  }
```

Comment:
It seems like clickModal already has a prevent default in it, and you're passing it into all of your components. I think you can use the this.props.clickModal so on line 68 just write onClick{this.props.clickModal(e)} (Same goes to all components)

Reply:
great catch. I will try this out!